api/v1alpha1/backstage_types.go

```diff
@@ -266,3 +258,7 @@ type TLS struct {
 func init() {
 	SchemeBuilder.Register(&Backstage{}, &BackstageList{})
 }
+
+func (s BackstageSpec) LocalDbEnabled() bool {
```

Comment:
nit: can we rename this into `IsLocalDbEnabled`? I think making functions returning a boolean sound like a yes/no question makes the intent clearer.

Reply:
+1